administrator/components/com_workflow/src/Helper/GraphHelper.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_workflow
+ *
+ * @copyright   (C) 2025 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Workflow\Administrator\Helper;
+
+use Joomla\CMS\Helper\ContentHelper;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * The first example class, this is in the same
+ * package as declared at the start of file but
+ * this example has a defined subpackage
+ *
+ * @since  _DEPLOY_VERSION_
+ */
+class GraphHelper extends ContentHelper
```

Comment:
Probably not needed at all, when you only extend? Then just use ContentHelper directly?